content/chainguard/libraries/java/global-configuration.md

```diff
@@ -23,7 +23,7 @@ Repository](https://www.sonatype.com/products/sonatype-nexus-repository). The
 repository manager acts as a single point of access for developers and
 development tools to retrieve the required libraries.
 
-At a high level, adopting the use of Chainguard Libraries consists of the the
+At a high level, adopting the use of Chainguard Libraries consists of the
```

Comment:
Move this into a separate PR to avoid conflicts.. also .. might be fixed already in main